recipes/act-atmos/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "act-atmos" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: af64e7b8a68bcd7e928b246c156042b6482f707decfcb56bd7b53847e9ec371a
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
This can be `noarch: python` if you remove this line and add `python >=3.6` below

Reply:
thanks for the suggestion, will update.  First time doing all this, so working through trying to get it right!